notebooks/notebooks/iclr2022/winrate-predictions.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.    plot(raw_white_loss, label='Self-play', color=(0.71372549, 0.84313725, 0.89411765))
</pre><p>nit: we're not using raw_black_loss here — does KataGo's selfplay loss look about the same regardless of which color it is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Yeah it's basically the same

Reply:
Just changed the `winrate-avg.pgf` cell so that it uses `np.concatenate([raw_white_loss, raw_black_loss])`